### ISSUE TITLE
Forked for significant refactoring

### ISSUE THREAD

**Issue by amilajack** [premii/hn#23]
I have forked this project and have already made significant contributions towards modernizing it. Here's the list of added support and improvements:
- [ ] ES6/7 support with babel
- [ ] Webpack support
- [ ] Extracted copy/pasted dependencies from source to npm dependencies
- [ ] ESLint support
- [ ] Travis CI support
- [x] Isomorphic Fetch API

I plan on adding / migrating to:
- [ ] React/Preact
- [ ] Mocha tests
- [ ] Sass support with postcss autoprefixer support, minification, concatenation
- [ ] Migration to localforage as an abstraction for localStorage

@premii It would be nice if you or other contributors can contribute to [my fork](https://github.com/amilajack/hn), which I hope will be merged later when finished

**Comment by thomasfl** [comment]
Sounds good. Lot's of things has happened since hn was realeased. The UI is very good and the codebase is rather small. Hope that won't change to much.

**Comment by amilajack** [comment]
None of the UI will change. The changes are only architectural changes and added tools. Adding react will probably reduce the amount of code and make it significantly easer to reason about.

**Comment by thomasfl** [comment]
Yes, the code should be much easier to read if rewritten with react. Do you plan to use the Touchstone UI library for react or create a new UI library? I have never used touchstone, but guess it can be tweaked to look like the HN app.

**Comment by amilajack** [comment]
I'm not really changing any of the html or css. I'm just refactoring to react, simplifying the logic of the application, and taking advantage of modern API's (such as fetch and maybe ServiceWorker in the future.)

**Comment by amilajack** [comment]
Also, do you have read/write access to the repo? Even if i make all these changes, I'm not sure if they will be merged because the repo owner doesn't seem to be active anymore.

**Comment by thomasfl** [comment]
Sorry, I don't have write rights. I've just used the hn app as a template app before I started to use ionic.
